paddle/trainer/NewRemoteParameterUpdater.cpp

```diff
@@ -50,8 +51,8 @@ void NewRemoteParameterUpdater::init(
 
   // create parameter server client.
   if (useEtcd_) {
-    parameterClient_ =
-        paddle_new_etcd_pserver_client((char *)pserverSpec_.c_str());
+    parameterClient_ = paddle_new_etcd_pserver_client(
+        (char *)pserverSpec_.c_str(), FLAGS_trainer_id == 0);
```

Comment:
Can you rebase / pull with latest develop? `paddle_new_etcd_pserver_client` now only takes one argument.